Configuration/PyReleaseValidation/scripts/das-selected-lumis.py

```diff
@@ -16,8 +16,8 @@
 lumi_data = lumi_data['data']
 
 def match_in(sub_list,lumi_list):
-  sub_list = map(int,sub_list)
-  lumi_list = map(int,lumi_list)
+  sub_list = list(map(int,sub_list))
```

Comment:
this starts to be a complicated way of writing
for i in range(int(sub_list[0]),int(sub_list[1])+1):
   if i >= int(lumi_list[0]) and i <= int(lumi_list[1]): return True
return False

Reply:
thanks for suggestion. I have updated it